package.json

```diff
@@ -28,7 +28,7 @@
     "keyv": "^3.0.0",
     "moment": "^2.21.0",
     "named-regexp": "^0.1.1",
-    "newrelic": "^3.0.0",
+    "newrelic": "^5.2.1",
```

Comment:
At the moment I think we don't have new relic enabled set up staging, which makes this double major version bump a bit risky. We should either stick to our current version or enable new relic on staging before deploying this.

Reply:
In this case this was the warning:

```
# Run  npm install newrelic@5.2.1  to resolve 1 vulnerability
SEMVER WARNING: Recommended action is a potentially breaking change
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ High          │ Denial of Service                                            │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ https-proxy-agent                                            │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ newrelic                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ newrelic > https-proxy-agent                                 │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/593                       │
└───────────────┴──────────────────────────────────────────────────────────────┘
```

How hard is it to enable newrelic in staging?

Reply:
As discussed through Slack, we are going to enable newrelic on staging.